### ISSUE TITLE
Publicly submitted issue

### ISSUE THREAD

**Issue by haudiobe** [Dash-Industry-Forum/ATSC#69]
Submitter: Wolfgang Schildbach
Email: wschi@dolby.com
We identified a bug in the DASH-IF IOP for ATSC 3.0. Specifically, the IOP prescribes using a value of “Descriptions” for the Accessibility@value element (tables 6, 7, 8). MPEG DASH, for the Role scheme defined, allows a value of “Description” (note the missing “s”).

This bug affects both AC-4 and MPEG-H.
